src/main/java/org/mockito/internal/stubbing/InvocationContainerImpl.java

```diff
@@ -128,8 +129,8 @@ public void clearInvocations() {
         registeredInvocations.clear();
     }
 
-    public List<StubbedInvocationMatcher> getStubbedInvocations() {
-        return stubbed;
+    public List<Stubbing> getStubbedInvocations() {
+        return (List) stubbed;
```

Comment:
We could certainly do that, thank you for the pointer! Problem is that it makes the public API a bit more complicated because the invariant wildcard leaks to the public interface.